akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowCompileSpec.scala

```diff
@@ -39,7 +39,7 @@ class FlowCompileSpec extends StreamSpec {
       "open3.run()" shouldNot compile
 
       val closedSource: Source[Int, _] = intSeq.via(open3)
-      "closedSource.run()" shouldNot compile
+      "closedSource.run()" should compile
```

Comment:
In this test we have to options:
1. Replace `shouldNot compile` to `should compile`
2. Remove these checks.
I am not sure what exactly is the purpose of this test so for now I chose the first option.
What do you think?

Reply:
I think it's fine to remove those tests - the method is exercised in SourceSpec, after all.

Reply:
Fixed

Reply:
Done